automatic-cluster-upgrades/README.md

```diff
@@ -1,6 +1,13 @@
 ---
 creation_date: 2021-09-17
+issues:
+- [https:/](https://github.com/giantswarm/giantswarm/issues/27420)
+last_review_date: 2024-02-23
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+- https://github.com/orgs/giantswarm/teams/team-honeybadger
 state: approved
+summary: As part of the Cluster API hackathon the idea of how to automate the cluster upgrades allows us to brainstorm how it can be achieved taking into account customer requirements and technical limitations brought under the new upstream implementation.
```

Comment:
> how to automate the cluster upgrades allows us to brainstorm how

i don't know what you're trying to say here - we brainstormed how to automated cluster upgrades?

Reply:
wow grammarly played on me. Yeah it was the intent